### ISSUE TITLE
Intersecting paths in some glyphs may cause improper rendering

### ISSUE THREAD

**Issue by thebodzio** [skosch/Crimson#10]
I noticed that some glyphs in this font are constructed from referenced glyph and path crossing that glyph. Unfortunately, this may cause problems with their rendering. As I checked, freetype2 seemd to render mentioned glyphs as intended, however Windows XP at some sizes displayed empty areas where the shapes intersected. Below is illustration of this issue.

![crimson-text-problem](https://cloud.githubusercontent.com/assets/1129853/5406362/74b40ca6-81b2-11e4-9717-4181edc6c19f.png)

So far I noticed this problem with the following glyphs: /oslash, /Oslas, /dcroat, /Dcroat, /lslash and /Lslash. Full list can be obtained thanks to FF's “Find problems…”. One way to fix this would be to simply “place” the reference in the given glyph and use “Remove Overlap”. I didn't do it myself because a) time, b) I didn't know if that's the smartest way to do it (don't know if, by a chance, FF can do that automatically when it generates binary file).

**Comment by NumericalEvolution** [comment]
I'm affraid the bug is still there, but only in OTF version.

![bug](https://cloud.githubusercontent.com/assets/12125847/9564630/c2d99996-4ead-11e5-8f66-36eaebc07f9f.png)

**Comment by katef** [comment]
I don't have a Windows machine, but I asked a friend to confirm this bug, using a fresh build of Crimson-Roman.otf from around ca492c6 or so. He says ł and Ł render fine in Windows 7.

@NumericalEvolution can you confirm that this still exists by generating an up to date OTF file? Which version of Windows are you using? I'd like to be able to reproduce this.

**Comment by thebodzio** [comment]
As for me, for day-to-day use I'm running Linux, but for the rare cases when I have to use Windows, I'm using an ancient XP. Anyway…

I've built OTF from ca492c6 and it appears to work all right in both Linux and XP. The version that didn't work (in Windows XP at least) was the one from the “Desktop Fonts” folder in the repo.

**Comment by adrientetar** [comment]
I have fixed some of the overlaps in the regular weight but they are numerous (so many diacritics were generated in the font).

**Comment by katef** [comment]
@adrientetar How do you fix that sort of thing? Is it just a case of changing the direction for the outline?

**Comment by skosch** [comment]
Fontforge has a validation tool, but I remember being unable to silence many of the warnings even after fixing what seemed to be the underlying problems, so I gave up eventually (this is a few years back). It's likely that there are still a whole bunch of reversed outlines, but I'm also on Linux and haven't had problems.

**Comment by adrientetar** [comment]
@katef Right-click on glyph > properties and you have a checkbox "remove overlap when generating"

**Comment by katef** [comment]
@adrientetar Thanks!
